### DISCUSSION TITLE
Tensorflow Support?

### DISCUSSION THREAD

**Discussion by mugeshk97** [MAIF/eurybia#17] [Q&A] [answered]
can we monitor the drift of tensorflow or any deep learning models?

**Comment by ThomasBouche** [comment, marked as answer]
Hi,
You can monitor the evolution of your data without giving your model. If your data are tabular. 

The model allows you to put into perspective the importance of data drift on the predictions of the model.
The features work if the model has a "predict_proba" method. 

we did not test the compatibility of tensorflow or deep learning models specifically
Are you using a tensorflow or deep learning model to make a classification or regression model on tabular data?

**Reply by mugeshk97** [reply]
tabular data as well as image data both
